index.js

```diff
@@ -345,6 +346,16 @@ function inspectNull(depth, opts) {
   return new InspectLeaf('null', styles.null, depth, opts)
 }
 
+function inspectCustomString(value, depth, opts) {
+  if (value.includes('\n')) {
+    value = value.replaceAll('\n', '\n' + '  '.repeat(depth))
+
+    opts.breakAlways = true
```

Comment:
```suggestion
    opts = { ...opts, breakAlways: true }
```

Otherwise the caller reference is mutated.